hexrd/rotations.py

```diff
@@ -178,18 +192,19 @@ def misorientation(q1, q2, *args):
 
     if len(args) == 0:
         # no symmetries; use identity
-        sym = (c_[1., 0, 0, 0].T, c_[1., 0, 0, 0].T)
+        sym = (np.c_[1.0, 0, 0, 0].T, np.c_[1.0, 0, 0, 0].T)
     else:
         sym = args[0]
```

Comment:
We shouldn't have an `args` argument. Instead, replace it with `sym` with a default value of `None`. `args` adds a layer of unnecessary obfuscation.

Reply:
I could, I don't really want to mess with a function that I haven't written a testcase for.  My PR was focused on the mathematical rotation transformation methods, this change is only here for formatting reasons.